apps/src/sites/studio/pages/devise/registrations/edit.js

```diff
@@ -63,6 +64,18 @@ $(document).ready(() => {
     );
   }
 
+  const schoolInformationMountPoint =
+    document.getElementById('school-information');
+  if (schoolInformationMountPoint) {
+    const store = getStore();
+    ReactDOM.render(
+      <Provider store={store}>
```

Comment:
What's redux needed for here?

Reply:
it's not used. I followed the convention I saw in the other mounted react components in case a future subcomponent needed to be connected, but yeah I can totally remove it!